### PR TITLE
Replaced  the links of 2 channels

### PR DESCRIPTION
**Pull request by Manipulativemachan**

### CODE REVIEW

File: streams/in.m3u [hunk 1]
```diff
@@ -409,7 +409,7 @@ http://wearelive.livebox.co.in/naxatratvhls/Naxatratv.m3u8
 #EXTINF:-1 tvg-id="Nazrana.in" status="online",Nazrana (720p) [Not 24/7]
 https://live.hungama.com/linear/nazrana/playlist.m3u8
 #EXTINF:-1 tvg-id="NCV.in" status="timeout",NCV (720p) [Not 24/7]
-http://103.146.174.60:1935/NCV/ncvstream/master.m3u8
+http://131.153.22.8:1935/NCV/ncvstream/chunklist_w509930903.m3u8
```
Comment (MapGuy11) [inline streams/in.m3u:412]: http://131.153.22.8:1935/NCV/ncvstream/playlist.m3u8

File: streams/in.m3u [hunk 2]
```diff
@@ -638,7 +638,7 @@ https://indo51.gcdn.co/MALYLAM-SuryaMusic/index.m3u8
 #EXTINF:-1 tvg-id="SuryaTV.in" status="error",Surya TV (720p) [Not 24/7]
 https://indo51.gcdn.co/MALYLAM-SuryaHD/index.m3u8
 #EXTINF:-1 tvg-id="Swantham.in" status="timeout",Swantham (720p) [Not 24/7]
-http://cdn1.logicwebs.in:1935/SWANTHAM/live/playlist.m3u8
+http://131.153.22.8:1935/SWANTHAM/live/chunklist_w436483673.m3u8
```
Comment (MapGuy11) [inline streams/in.m3u:641]: http://131.153.22.8:1935/SWANTHAM/live/playlist.m3u8

File: streams/in.m3u (hunk 1 above)
Comment (MapGuy11) [inline streams/in.m3u:412]: http://131.153.22.8:1935/NCV/ncvstream/playlist.m3u8

File: streams/in.m3u (hunk 2 above)
Comment (MapGuy11) [inline streams/in.m3u:641]: http://131.153.22.8:1935/SWANTHAM/live/playlist.m3u8

**Review by MapGuy11 — CHANGES_REQUESTED** [review]

**Comment by MapGuy11** [comment]
@Manipulativemachan are you going to fix this?

**Comment by Manipulativemachan** [comment]
> @Manipulativemachan are you going to fix this?

Please make the changes.

File: streams/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [inline streams/in.m3u:412]:
```suggestion
http://131.153.22.8:1935/NCV/ncvstream/playlist.m3u8
```

File: streams/in.m3u (hunk 2 above)
Comment (LaneSh4d0w) [inline streams/in.m3u:641]: http://131.153.22.8:1935/SWANTHAM/live/playlist.m3u8

Comment (LaneSh4d0w) [inline streams/in.m3u:641]:
```suggestion
http://131.153.22.8:1935/SWANTHAM/live/playlist.m3u8
```

**Review by BellezaEmporium — APPROVED** [review]

**Review by its0x08 — APPROVED** [review]

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**